build.gradle

```diff
@@ -59,7 +59,7 @@ buildscript {
                 'ktlint': '0.44.0'
         ]
 
-        javaVersion = JavaVersion.VERSION_1_8
+        javaVersion = JavaVersion.VERSION_11
```

Comment:
❌ This has practical implications and it's easy to end up with code that won't run on older Android versions.

Reply:
AGP will desugar them.

Reply:
As far as I can tell, only if you also use `coreLibraryDesugaring`. We've been holding off because a) we don't really need it for our code and b) it didn't work with commons-io because of https://issuetracker.google.com/issues/160484830

The last time I checked AGP didn't use an r8 version that includes a fix for the issue mentioned above.

Reply:
84f9fe1f07a117e546e254dbef6d0fe407aeb92b

Reply:
84f9fe1f07a117e546e254dbef6d0fe407aeb92b